python/ray/experimental/state/api.py

```diff
@@ -40,100 +47,128 @@ def _list(resource_name: str, options: ListApiOptions, api_server_url: str = Non
     r.raise_for_status()
 
     response = r.json()
-    if not response["result"]:
+    if response["result"] is False:
         raise ValueError(
             "API server internal error. See dashboard.log file for more details."
         )
+
+    if _print_api_stats:
```

Comment:
How do we expect the python api to be used? If we just print out errors, it's hard for the caller to do error handling (e.g. retry) since they don't know whether error happens or not programmatically?

Reply:
Probably we can see how error handling is done on the product side for anyscale apis.

Reply:
I think this can answer this question ! https://github.com/ray-project/ray/pull/24413#discussion_r879659312